### ISSUE TITLE
Left 4 Dead 2 - Last Stand Map

### ISSUE THREAD

**Issue by nitro-1** [ValveSoftware/Source-1-Games#3434]
Hello, 
The new Campaign The Last Stand First Map : c14m1_junkyard
There is a Bug with Generators, The Last Generator Will Not Work After Triggering The Other Generators. The Trigger Bug almost everytime in that map, and those generators are necessary to proceed on the map.

Modded Coop Server Hosted On Linux Debian 10.

**Comment by Tsuey** [comment]
I don't think this is still an issue? Noticed it's a deleted account, with server mods used, and a gamebreaking bug, which would have been reported again by now. Closest (but still distant) persisting issue we know about is https://github.com/Tsuey/L4D2-Community-Update/issues/22 which is just a visual bug that doesn't prevent completion.

**Comment by kisak-valve** [comment]
Closing per the last comment and because the reporting user has turned into a ghost.
